includes/Admin/AnyCommentGenericSettings.php

```diff
@@ -524,33 +524,33 @@ public function init_settings () {
 		$form->add_section(
 			$this->section_builder()
 			     ->set_id( 'elements' )
-			     ->set_title( __( 'Elements', "anycomment" ) )
+			     ->set_title( __( 'Common Elements', "anycomment" ) )
 			     ->set_fields( [
 				     $this->field_builder()
 				          ->select()
 				          ->set_id( self::OPTION_FORM_TYPE )
-				          ->set_title( __( 'Comment form', "anycomment" ) )
+				          ->set_title( __( 'Who can comment?', "anycomment" ) )
 				          ->set_args( [
 					          'options' => [
-						          self::FORM_OPTION_ALL            => __( 'Anyone', 'anycomment' ),
-						          self::FORM_OPTION_WORDPRESS_ONLY => __( 'WordPress only', 'anycomment' ),
-						          self::FORM_OPTION_SOCIALS_ONLY   => __( 'Socials only', 'anycomment' ),
-						          self::FORM_OPTION_GUEST_ONLY     => __( 'Guests only', 'anycomment' ),
+						          self::FORM_OPTION_ALL            => __( 'WordPress users, Social Network Users, as well as public and anonymous guests.', 'anycomment' ),
+						          self::FORM_OPTION_WORDPRESS_ONLY => __( 'WordPress users only', 'anycomment' ),
+						          self::FORM_OPTION_SOCIALS_ONLY   => __( 'Social Network users only', 'anycomment' ),
+						          self::FORM_OPTION_GUEST_ONLY     => __( 'Any public or anonymous guests', 'anycomment' ),
 					          ],
 				          ] )
-				          ->set_description( esc_html( __( 'Users who able to leave comments.', "anycomment" ) ) ),
+				          ->set_description( esc_html( __( 'Which kind of users should be able to leave comments?', "anycomment" ) ) ),
 
 				     $this->field_builder()
 				          ->text()
 				          ->set_id( self::OPTION_GUEST_FIELDS )
-				          ->set_title( __( 'Guest Fields', "anycomment" ) )
-				          ->set_description( esc_html( __( 'Use this rearrange guest form fields or remove something. {name} is required and if you do not add it, it will be added by plugin. {name} is name field, {email} is email field, {website} is website field.', "anycomment" ) ) ),
+				          ->set_title( __( 'Comment form fields', "anycomment" ) )
```

Comment:
We have titles capitalized first letter, maybe we could make it the same way here, so that we won't need to fix it for many languages later

Reply:
```suggestion
				          ->set_title( __( 'Comment Form Fields', "anycomment" ) )
```